canvas_modules/common-canvas/__tests__/common-properties/components/validation-message-test.js

```diff
@@ -34,116 +49,127 @@ const infoMessage = {
 
 describe("validation-message renders correctly", () => {
 	it("props should have been defined", () => {
-		const wrapper = mount(
+		render(
 			<ValidationMessage
 				messageInfo={warningMessage}
 				state="hidden"
 				inTable
 				tableOnly
 			/>
 		);
-		expect(wrapper.prop("messageInfo")).to.equal(warningMessage);
-		expect(wrapper.prop("state")).to.equal("hidden");
-		expect(wrapper.prop("inTable")).to.equal(true);
-		expect(wrapper.prop("tableOnly")).to.equal(true);
+		expectJest(mockValidationMessage).toHaveBeenCalledWith({
+			"messageInfo": warningMessage,
+			"state": "hidden",
+			"inTable": true,
+			"tableOnly": true
+		});
 	});
 	it("validation-message renders error message correctly", () => {
-		const wrapper = mount(
+		const wrapper = render(
 			<ValidationMessage
 				messageInfo={errorMessage}
 			/>
 		);
-		const messageWrapper = wrapper.find("div.properties-validation-message");
+		const { container } = wrapper;
+		const messageWrapper = container.getElementsByClassName("properties-validation-message");
 		expect(messageWrapper).to.have.length(1);
-		expect(messageWrapper.find("div.icon svg.canvas-state-icon-error")).to.have.length(0);
-		expect(messageWrapper.find("span").text()).to.equal(errorMessage.text);
+		expect(messageWrapper[0].getElementsByClassName("canvas-state-icon-error")).to.have.length(0);
+		expect(messageWrapper[0].querySelector("span").textContent).to.equal(errorMessage.text);
 	});
 	it("validation-message renders error message correctly in table", () => {
-		const wrapper = mount(
+		const wrapper = render(
 			<ValidationMessage
 				messageInfo={errorMessage}
 				inTable
 			/>
 		);
-		const messageWrapper = wrapper.find("div.properties-validation-message");
+		const { container } = wrapper;
+		const messageWrapper = container.getElementsByClassName("properties-validation-message");
 		expect(messageWrapper).to.have.length(1);
-		expect(messageWrapper.find("div.icon svg.canvas-state-icon-error")).to.have.length(1);
-		expect(messageWrapper.find("span").text()).to.equal(errorMessage.text);
+		expect(messageWrapper[0].getElementsByClassName("canvas-state-icon-error")).to.have.length(1);
+		expect(wrapper.getByText(/Testing validation message/).textContent).to.equal(errorMessage.text);
 	});
 	it("validation-message renders warning message correctly", () => {
-		const wrapper = mount(
+		const wrapper = render(
 			<ValidationMessage
 				messageInfo={warningMessage}
 			/>
 		);
-		const messageWrapper = wrapper.find("div.properties-validation-message");
+		const { container } = wrapper;
+		const messageWrapper = container.getElementsByClassName("properties-validation-message");
 		expect(messageWrapper).to.have.length(1);
-		expect(messageWrapper.find("div.icon svg.canvas-state-icon-warning")).to.have.length(0);
-		expect(messageWrapper.find("span").text()).to.equal(warningMessage.text);
+		expect(within(messageWrapper[0]).queryAllByRole("svg")).to.have.length(0);
```

Comment:
can we replicate `expect(messageWrapper.find("div.icon svg.canvas-state-icon-warning")).to.have.length(0);` in RTL ?

Reply:
I changed it to `messageWrapper[0].querySelectorAll("svg.canvas-state-icon-warning")`